doc/generic/pgf/text-en/pgfmanual-en-library-matrices.tex

```diff
@@ -12,7 +12,8 @@ \section{Matrix Library}
 
 \begin{tikzlibrary}{matrix}
     This library package defines additional styles and options for creating
-    matrices.
+    matrices. (The basic matrix styles and options can be found in
+    Section~\ref{section-matrices}.)
```

Comment:
This sentence is important enough to stand without parentheses.

Reply:
done